baseapp-profiles/baseapp_profiles/graphql/object_types.py

```diff
@@ -126,10 +128,23 @@ def resolve_metadata(cls, instance, info):
         return ProfileMetadata(instance, info)
 
     @classmethod
-    def resolve_members(cls, instance, info, **kwargs):
+    def resolve_members(cls, instance, info, order_by_status=None, **kwargs):
         if not info.context.user.has_perm("baseapp_profiles.view_profile_members", instance):
             return instance.members.none()
-        return instance.members.all()
+
+        members_queryset = instance.members.all()
+
+        if order_by_status == "custom":
```

Comment:
Ideally the filters should be on a FilterSet class that the ObjectType uses. The ProfileObjectType have an example. By doing that any DjangoFilterConnection that points to that ObjectType will have those filters without the need to define them on every connection